modules/ROOT/nav.adoc

```diff
@@ -77,16 +77,16 @@ include::{partialsdir}/attributes.adoc[]
 
 
 
-** xref:security.adoc[Security]
-*** xref:app-security.adoc[{app-security-service}]
-**** xref:app-security.adoc#introduction[Introduction]
-**** xref:app-security.adoc#setup[Setting Up]
-**** xref:app-security.adoc#monitoring[Monitoring App Security]
-*** xref:device-security.adoc[{device-security-service}]
-**** xref:device-security.adoc#introduction[Introduction]
-**** xref:device-security.adoc#setup[Setting Up]
-**** xref:device-security.adoc#device-trust[Performing Device Trust Checks]
-**** xref:device-security.adoc#cert-pinning[Certificate Pinning]
+** xref:security.adoc[Mobile Security]
```

Comment:
![Screenshot 2019-07-03 at 09 38 43](https://user-images.githubusercontent.com/6498727/60576980-63f1bf00-9d76-11e9-89b4-7ea048db1b30.png)

Both the root of this nav and the first option are `Mobile Security` now.